src/Popper.js

```diff
@@ -152,25 +160,70 @@ export class Popper extends Component<PopperProps, PopperState> {
         this.updatePopperInstance();
       }
     }
+
+    if (this.props.subscribe !== prevProps.subscribe) {
+      this.unsubscribe();
+      this.setState({
+        unsubscribe: this.props.subscribe(this),
+      });
+    }
+  }
+
+  onUpdate() {
+    this.state.subscribers.forEach(popper =>
+      popper.state.popperInstance.update()
```

Comment:
this should be `scheduleUpdate`

Reply:
When I used `scheduleUpdate` there was a delay. `update` fixed it.

Reply:
When I used `scheduleUpdate` there was a delay. `update` fixed it.